app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+
+[See it on Plunker](http://plnkr.co/edit/MRN9blKg6A3w8G0RkyJD?p=preview)
+
+```html
+<dom-module id="x-foo">
+    <template>
+        <h1><slot name='heading1'></slot></h1>
+        <p><slot name='para'></slot></p>
+    </template>
+</dom-module>
+
+<x-foo>
+    <div slot="heading1">Heading!</div>
+    <div slot="para">Paragraph text.</div>
+</x-foo>
+```
+
+To style slotted content, use the `::slotted()` syntax.
+
+`::slotted(*)` selects all slotted content:
+
+[See it on Plunker](http://plnkr.co/edit/cGrbntuantGtqj7Poqft?p=preview)
+
+```html
+<dom-module id="x-foo">
+    <template>
+      <style>
+        ::slotted(*) {
+          font-family: sans-serif;
+          color:green;
+        }
+      </style>
+        <h1>
+            <div><slot name='heading1'></slot></div>
+        </h1>
+        <p>
+            <slot name='para'></slot>
+        </p>
+    </template>
+</dom-module>
+<x-foo>
+    <div slot="heading1">Heading 1. I'm green.</div>
+    <div slot="para">Paragraph text. I'm green too.</div>
+</x-foo>   
+```
+
+[See it on Plunkr](http://plnkr.co/edit/suVhRDAB5Rq7Q7T6zbMI?p=info)
+
+You can select by element type:
+
+```html
+<dom-module id="x-foo">
+	<template>
+		<style>
+		::slotted(h1) {
+			font-family: sans-serif;
+			color:green;
+		}
+		::slotted(p) { 
+			font-family: sans-serif;
+          color:blue;
+      }
+      </style>    
+            <slot name='heading1'></slot>
+            <slot name='para'></slot>
+    </template>
+</dom-module>
+<x-foo>
+    <h1 slot="heading1">Heading 1. I'm green.</h1>
+    <p slot="para">Paragraph text. I'm blue.</p>
+</x-foo>
+```
+
+You can select by class:
+
+[See it on Plunker](http://plnkr.co/edit/Ep8AVOHgiwQjtv8x5kwd?p=preview)
+
+```
+<dom-module id="x-foo">
+    <template>
+      <style>
+        ::slotted(.green) {
+          color:green;
+        }
+      </style>
+        <p>
+            <slot name='para1'></slot>
+        </p>
+        <p>
+            <slot name='para2'></slot>
+        </p>
+    </template>
+</dom-module>
+
+<x-foo>
+    <div slot="para1" class="green">I'm green!</div>
+    <div slot="para1">I'm not green.</div>
+    <div slot="para2" class="green">I'm green too.</div>
+    <div slot="para2">I'm not green.</div>
+</x-foo> 
+```
+
+And you can select by slot name:
+
+[See it on Plunker](http://plnkr.co/edit/FnFLnownBJc0Ur8vC6dd?p=preview)
+
+```
+<dom-module id="x-foo">
+    <template>
+      <style>
+        slot[name='para1']::slotted(*) {
+          color:green;
+        }
+      </style>
+        <p>
+            <slot name='para1'></slot>
+        </p>
+        <p>
+            <slot name='para2'></slot>
+        </p>
+    </template>
+</dom-module>
+
+<x-foo>
+    <div slot="para1">I'm green!</div>
+    <div slot="para2">I'm not green.</div>
+</x-foo> 
+```
+
+For more examples, see [Rob Dodson's Shadow DOM CSS Cheat Sheet](http://robdodson.me/shadow-dom-css-cheat-sheet/).
```

Comment:
Not sure this link is worth it... Not a lot of examples and some are likely to be confusing, like host-context which we don't support.  (also, the statements about browser support are really out of date.)